modules/react/src/utils/position-children-under-views.js

```diff
@@ -50,8 +52,26 @@ export default function positionChildrenUnderViews({children, viewports, deck})
       viewState
     });
 
-    const style = {position: 'absolute', left: x, top: y, width, height};
+    const style = {
+      position: 'absolute',
+      zIndex: childStyle && childStyle.zIndex,
+      pointerEvents: 'none',
```

Comment:
Why you need set `pointerEvents` to `none`?

Reply:
If this container is on top of the map it will block pointer input to the deck canvas.